src/10-ipfs-0-4-0-released/index.md

```diff
@@ -0,0 +1,121 @@
+---
+date: 2016-03-17
+id: 10-ipfs-0-4-0-released
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+- {name: "10-ipfs-0-4-0-released", link: "./" }
+tags: modules
+title: IPFS 0.4.0 has been released
+author: Kyle Drake and Whyrusleeping
+collection: posts
+---
+
+[IPFS 0.4.0](https://github.com/ipfs/go-ipfs/releases/tag/v0.4.0) has been
+released! Among the many changes are a revamped implementation of the IPFS
+communication protocols, increased performance, improvements to IPNS, many
+bugfixes, and a couple new features to make IPFS even more powerful.
+
```

Comment:
There is no section of 'awesome new things'. Saying IPNS resolution is faster by X % and how adding files to IPFS is faster too by Y % gives people a more easier way to quantify the impact of this PR.